base_tier_validation_escalate/models/tier_review.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class TierReview(models.Model):
+    _inherit = "tier.review"
+    _order = "sequence"
+
+    name = fields.Char(compute="_compute_definition_data", store=True, readonly=False)
+    status = fields.Selection(selection_add=[("escalated", "Escalated")],)
+    review_type = fields.Selection(
+        compute="_compute_definition_data", store=True, readonly=False,
+    )
+    reviewer_id = fields.Many2one(
+        comodel_name="res.users",
+        compute="_compute_definition_data",
+        store=True,
+        readonly=False,
+    )
+    reviewer_group_id = fields.Many2one(
+        comodel_name="res.groups",
+        compute="_compute_definition_data",
+        store=True,
+        readonly=False,
+    )
+    sequence = fields.Float()
+    has_comment = fields.Boolean(
+        compute="_compute_definition_data", store=True, readonly=False,
+    )
+    approve_sequence = fields.Boolean(
+        compute="_compute_definition_data", store=True, readonly=False,
+    )
+
+    @api.depends(
+        "definition_id.name",
+        "definition_id.review_type",
+        "definition_id.reviewer_id",
+        "definition_id.reviewer_group_id",
+        "definition_id.has_comment",
+        "definition_id.approve_sequence",
+    )
+    def _compute_definition_data(self):
```

Comment:
I see this is overwriting related behavior from Tier Validation Base module, @LoisRForgeFlow what do you think to move from related to compute + stored in base module? @kittiu why did you apply this fields behavior change.

Reply:
@HviorForgeFlow  So far i will need some persistant fields to create a new adhoc tier (escalate). But with the existing it does not allowed me to do (now everything tie to definition). Also dont like it too, but see no other way yet.

Reply:
It seems not ideal indeed. However, if this is unavoidable (I would let @kittiu to decide it), I would only ask to add some comments in the code indicating that most of this is overriding base module and maybe also another comment in the readme highlighting that the data model is changed when installing this module.

Reply:
Thanks @LoisRForgeFlow also what i will try to do is to make sure model is back to normal when unistalled.
I shall go ahead.

Reply:
Set model back to original.
https://github.com/ecosoft-odoo/server-ux/blob/13.0-add-base_tier_validation_escalate/base_tier_validation_forward/hooks.py

Note: the forwarded tier will be set back to status approved, and the sequence will be round down to its integer.